sdk/cosmos/azure-cosmos-encryption/src/test/java/com/azure/cosmos/encryption/EncryptionSyncApiCrudTest.java

```diff
@@ -282,20 +383,20 @@ public void batchExecution() {
 
     @Test(groups = {"encryption"}, timeOut = TIMEOUT)
     public void batchExecutionWithOptionsApi() {
-        String itemId= UUID.randomUUID().toString();
+        String itemId = UUID.randomUUID().toString();
         EncryptionPojo createPojo = getItem(itemId);
-        EncryptionPojo replacePojo =  getItem(itemId);
+        EncryptionPojo replacePojo = getItem(itemId);
         replacePojo.setSensitiveString("ReplacedSensitiveString");
         CosmosBatch cosmosBatch = CosmosBatch.createCosmosBatch(new PartitionKey(itemId));
         CosmosBatchItemRequestOptions cosmosBatchItemRequestOptions = new CosmosBatchItemRequestOptions();
 
         cosmosBatch.createItemOperation(createPojo, cosmosBatchItemRequestOptions);
-        cosmosBatch.replaceItemOperation(itemId, replacePojo,cosmosBatchItemRequestOptions);
+        cosmosBatch.replaceItemOperation(itemId, replacePojo, cosmosBatchItemRequestOptions);
         cosmosBatch.upsertItemOperation(createPojo, cosmosBatchItemRequestOptions);
         cosmosBatch.readItemOperation(itemId, cosmosBatchItemRequestOptions);
         cosmosBatch.deleteItemOperation(itemId, cosmosBatchItemRequestOptions);
 
-        CosmosBatchResponse batchResponse = this.cosmosEncryptionContainer.executeCosmosBatch(cosmosBatch);
```

Comment:
Now do we have tests for all four bulk overloads , 2 for async and 2 for sync ?

Reply:
Yes we have